app/controllers/application_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  after_action  :change_dog
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.